src/current/v23.2/live-migration-service.md

```diff
@@ -0,0 +1,612 @@
+| `SSL_CA`      | Mount path to the CA certificate for the LMS.                                                                                  |
+| `SSL_CERT`    | Mount path to the certificate for the LMS.                                                                                     |
+| `SSL_KEY`     | Mount path to the key for the LMS.                                                                                             |
+| `CA_TLS_CERT` | Mount path to the CA certificate for the orchestrator. This is optional and is mainly used if you self-sign your certificates. |
+| `TLS_CERT`    | Mount path to the certificate for the orchestrator.                                                                            |
+| `TLS_KEY`     | Mount path to the key for the orchestrator.                                                                                    |
+
+{{site.data.alerts.callout_success}}
+Cockroach Labs recommends mounting certificates to `/app/certs`.
+{{site.data.alerts.end}}
+
+#### Service type
+
+~~~ yaml
+lms:
+  service:
+    type: ClusterIP
+    port: 9043
+    metricsPort: 9044
+...
+orchestrator:
+  service:
+    type: ClusterIP
+    port: 4200
+    metricsPort: 4201
+~~~
+
+`service` specifies the [Kubernetes service type](https://kubernetes.io/docs/concepts/services-networking/service/#publishing-services-service-types) and ports for the LMS instances and orchestrator.
+
+#### Prometheus Operator
+
+~~~ yaml
+serviceMonitor:
+  enabled: false
+  labels: {}
+  annotations: {}
+  interval: 30s
+  namespaced: false
+~~~
+
+`serviceMonitor` is a custom resource used with the [Prometheus Operator](https://github.com/prometheus-operator/prometheus-operator/tree/main) for monitoring Kubernetes. For more information, see the [Prometheus Operator documentation](https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/user-guides/getting-started.md).
+
+## Security
+
+{{site.data.alerts.callout_danger}}
+Cockroach Labs **strongly** recommends the following:
+
+- Manage your LMS configuration in an [external Kubernetes secret](#manage-external-secret).
+- To establish secure connections between the LMS pods and with your client, set up TLS certificates for the [LMS](#configure-lms-certificate), [source database and CockroachDB](#configure-source-and-target-certificates), and [orchestrator and client](#configure-orchestrator-and-client-certificates).
+{{site.data.alerts.end}}
+
+#### Manage external secret
+
+Use an external secrets manager such as [External Secrets Operator](https://external-secrets.io/latest/) to create and manage a Kubernetes secret containing your LMS configuration. For information on Kubernetes secrets, see the [Kubernetes documentation](https://kubernetes.io/docs/concepts/configuration/secret/).
+
+In the [LMS configuration](#configuration), specify the name of your external secret with `configSecretName`:
```

Comment:
You might want to add a note that this will map 1:1 with the `target name` you specify in the external secret?

You also want to mention that these will be two distinct config secret names because the values that the LMS and orchestrator use are different.